### PR TITLE
Update accelerated-vpn.md

### PR DESCRIPTION
**Pull request by ifaqeer-zz**
1) Added for clarity that one GA from a separate IP pool is created per VPN tunnel and
2) These GA are not visible in the console and are only available for use for VPN tunnels. This is to avoid any confusion as many users think that just because accelerated VPN create two GAs, a) they are visible in the console and b) can be used like other normal GA functions, which is not true/not the case.

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: doc_source/accelerated-vpn.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
...
 You can optionally enable acceleration for your Site\-to\-Site VPN connection\. An accelerated Site\-to\-Site VPN connection \(accelerated VPN connection\) uses AWS Global Accelerator to route traffic from your on\-premises network to an AWS edge location that is closest to your customer gateway device\. AWS Global Accelerator optimizes the network path, using the congestion\-free AWS global network to route traffic to the endpoint that provides the best application performance \(for more information, see [AWS Global Accelerator](https://aws.amazon.com/global-accelerator/)\)\. You can use an accelerated VPN connection to avoid network disruptions that might occur when traffic is routed over the public internet\.
 
-When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel\.
+When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel\. Note: each tunnel uses a separate accelerator and a separate pool of IP addresses for the tunnel endpoint IP addresses. These accelerators are not visible in the AWS Console (AWS Global Accelerator->Accelerators page).
```
Comment (katerini) [inline doc_source/accelerated-vpn.md]: Hi ifaqeer - the information about using a separate pool of IP addresses is already on this page, a bit further down. Is your intention to move this info further up?

**Comment by ifaqeer-zz** [comment]
Hi @katerini a) I re-wrote the sentence to make it bit more clear vs the way it it originally written is vague b) More importantly, the missing piece is the fact that the two accelerators that are created and used by the VPN tunnels, are not visible externally in the Console. The way it come across if I were to read is that there are two accelerator(GA) created by enabling this VPN feature, and if I go to to the Console, I should see these accelerators there and use it. That mean, I get the two accelerators with their static IPs, and I can use the same two accelerators for other things beside the VPN, which is not the case here. To summarize, VPN service creates these two accelerators, but do not expose them in the console as it would when you create GA normally and this is not called out here. Thanks

**Comment by katerini** [comment]
Hi @ifaqeer - apologies that I wasn't clearer in my comment. I do agree about calling out that you can't view the accelerators in the console. I was referring specifically to the statement "separate pool of IP addresses for the tunnel endpoint IP addresses" which is already explained further down the page. If we add it here, it will be repeated twice on the page.

If the main issue is to call out that the accelerators are not viewable/accessible, then perhaps we can write the sentence as follows:
'When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel. You cannot view or manage these accelerators yourself by using the AWS Global Accelerator console or APIs.'

**Comment by ifaqeer-zz** [comment]
Hi @katerini 

> apologies that I wasn't clearer in my comment. I do agree about calling out that you can't view the accelerators in the console. I was referring specifically to the statement "separate pool of IP addresses for the tunnel endpoint IP addresses" which is already explained further down the page. If we add it here, it will be repeated twice on the page.

==> I think we both are saying the same thing but in a slightly different way. Let's keep this sentence as-is 👍 

> If the main issue is to call out that the accelerators are not viewable/accessible, then perhaps we can write the sentence as follows:
> 'When you create an accelerated VPN connection, we create and manage two accelerators on your behalf, one for each VPN tunnel. You cannot view or manage these accelerators yourself by using the AWS Global Accelerator console or APIs.'

==> Yes, this is the main call out that is missing.  Thank you

**Comment by katerini** [comment]
Great, I've modified the sentence as discussed, and merged. Thanks!

**Pull request merged**